index.html

```diff
@@ -1771,6 +1771,50 @@ <h3>Handling changes to worker status</h3>
   <h2>
     Security and privacy considerations
   </h2>
+  <section>
+    <h3>Own contribution estimate</h3>
+    <p>
+      The <dfn>own contribution estimate</dfn> provides a percentage-based indication of how much the current
+      site contributes to the global system pressure. This insight helps developers determine whether
+      a critical or serious pressure state originates primarily from their own application or from
+      external sources.
+    </p>
+    <p>
+      <em>If the majority of the pressure comes from external sources</em>, the site may choose to simplify
+      its features to maintain a smooth experience. It may also prompt users to close other
+      applications to reduce overall system pressure.
```

Comment:
This doesn't seem unique to pressure from external sources. Surely the site would simplify its features if it itself was exceeding the capabilities of the device, since even an admonition to close other applications would subject the user to poor performance up until the point they broke down and started looking at what else they might have open?